packages/valory/skills/market_creation_manager_abci/behaviours.py

```diff
@@ -1932,7 +1932,7 @@ def _prepare_ask_question_mstx(
         return {
             "to": self.params.realitio_contract,
             "data": response.state.body["data"],
-            "value": ETHER_VALUE,
+            "value": self.params.realitio_answer_question_bounty,
```

Comment:
This change is not reflected here:

https://github.com/valory-xyz/market-creator/blob/3f98888043e56aac773817f4c3bf96f835d16607/packages/valory/skills/market_creation_manager_abci/behaviours.py#L395

Reply:
@0xArdi please attach an example from the Mech.

Reply:
Thats not necessary. Example from the mech: 

Multisend instruction [with value](https://github.com/valory-xyz/mech/blob/main/packages/valory/skills/task_submission_abci/behaviours.py#L451), and prepared safe transaction with [no value](https://github.com/valory-xyz/mech/blob/main/packages/valory/skills/task_submission_abci/behaviours.py#L849).

[A tx](https://gnosisscan.io/tx/0x1736c82007ada16e74757844eccebdde0a18a62888dd48d532bf5280d0645606) from the code above.